spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/EscapeCharacter.java

```diff
@@ -31,7 +30,7 @@
 public final class EscapeCharacter {
 
 	public static final EscapeCharacter DEFAULT = EscapeCharacter.of('\\');
-	private static final List<String> TO_REPLACE = Arrays.asList("_", "%");
+	private static final List<String> TO_REPLACE = List.of("_", "%");
```

Comment:
We have many more occurences where we could use `List.of(…)` or `new HashSet<>(Arrays.asList(…))`. Care to have a look at these as well? Merging just one line change without considering the other places takes us more time than value it creates.

Reply:
I really didn’t think it through. Thank you for giving me good advice.